src/faq/faqs.njk

```diff
@@ -13,7 +13,10 @@ eleventyComputed:
 ---
 
 <article class="section-card">
-  <h1 class="section-header">{{ faq.question | markdownInline | safe }}</h1>
+  <h1 class="section-header">
+    {{ faq.question | markdownInline | safe }}
+    {% include "components/badges/freshness.njk" %}
```

Comment:
The components really belong outside of the `<h1>`. Maybe you'll need to add a `<header>`  around it to make it work.